### PR TITLE
Adding SX60 work by amnobis and configurator settings

### PR DESCRIPTION
**Pull request by amnesia0287**
The actual SX60 work was done by @amnobis who made the sx60. I just cleaned it up and added support for the configurator (at least I'm pretty sure).

### CODE REVIEW

File: keyboards/sx60/keymaps/amnobis/keymap.c [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#include "sx60.h"
```
Comment (mechmerlin) [inline keyboards/sx60/keymaps/amnobis/keymap.c]: Can you change this to be `#include QMK_KEYBOARD_H`

File: keyboards/sx60/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#include "sx60.h"
```
Comment (mechmerlin) [inline keyboards/sx60/keymaps/default/keymap.c]: Can you change this to be `#include QMK_KEYBOARD_H`

**Comment by amnesia0287** [comment]
Requested changes completed.

File: .vscode/settings.json [hunk 3]
```diff
@@ -12,6 +12,7 @@
         "*.c": "c",
         "*.cpp": "cpp",
         "*.hpp": "cpp",
-        "xstddef": "c"
+        "xstddef": "c",
+        "iosfwd": "c"
```
Comment (drashna) [inline .vscode/settings.json]: Could you revert the changes here?

File: keyboards/sx60/readme.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+SX60
+===
+
+![SX60](https://i.imgur.com/hZZHrRr.jpg)
+
+Keyboard Maintainer: [amnobis](https://github.com/amnobis)
```
Comment (drashna) [inline keyboards/sx60/readme.md]: Could you make sure that these three lines have double spaces at the end of the lines, so that they are parsed correctly?

Otherwise, they'll end up on one line.

**Comment by drashna** [comment]
Awesome!  Looks good!
